test/e2e/redux/reducers/register.reducer.spec.ts

```diff
@@ -22,6 +22,7 @@ const initalState: IRegisterState = {
     withdrawalMnemonic: "",
     withdrawalVerification: false,
     withdrawalKey: "",
+    network: "",
```

Comment:
maybe prysm as default value?

Reply:
It's default in the dropdown and will set it if nothing is selected. Idk, don't want to hardcode on many places 🤷‍♀

Reply:
It would make much more sense to set default here instead of that specific container